golang/examples/vout.go

```diff
@@ -38,7 +42,7 @@ func ParseVout(vout []byte) string {
 		voutData := prettifyOutput(numOutput, payload, value, outputType)
 
 		// Concat vout information onto `outputs`
-		outputs = outputs + "\n" + voutData
+		outputs = outputs + voutData
```

Comment:
names could be improved here. `outputs` could be something like `formattedOutputs` and `voutData` isn't actually about the `vout` at all. it's a single formatted output

same in `ParseVin`

Reply:
Would renaming `voutData` to simply `data` suffice?

Reply:
Or `outputData`?